custom/icds_reports/reports/reports.py

```diff
@@ -20,56 +20,52 @@
 @location_safe
 class MPRReport(IcdsBaseReport):
 
-    title = '1. Identification and Basic Information'
+    title = 'MPR Report'
```

Comment:
nit: does MPR Report mean "Monthly Progress Report Report" ? (similar for ASR Report later)